wiki/quest-modding.md

```diff
@@ -128,15 +126,10 @@ Developer Mode on your Quest.
 Before modding, run Beat Saber once, play a level and immediately fail!
 :::
 
-Open bugjaeger on your Phone and connect your Quest.
+Open Sidequest on your Phone connect and your Quest with your phone via the USB Cord.
 You should get a USB debugging pop-up in your Quest and on your phone. Select allow on both devices and if you prefer,
 select always allow.
-Once bugjaeger picks your Quest up, install the BMBF APK by doing following:
-
-![installAPKusingPhone.png](~@images/beginners-guide/InstallAPK.png)
-
-After you pressed ok, allow file access and select the download APK file which should be labeled `com.weloveoculus.BMBF.apk`.
-The apk file should now install to your Quest.
+Now if SideQuest picked your Quest up you should see the SideQuest logo with a green dot next to it at the bottom of your Screen. Then install BMBF by clicking the search icon at the top of SideQuest and search for `BMBF`. You should see an app with an unicorn as symbol. Tap on that and click `INSTALL TO HEADSET`. Afterwards you should have BMBF installed on your Quest and are ready to mod Beat Saber.
```

Comment:
Linting issue. Line is too long at 404 chars. Max is 120.
Not sure why this got past the PR check.